src/index.js

```diff
@@ -1,10 +1,10 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from "react";
```

Comment:
I introduced `Prettier` but didn't commit it yet. I can follow up with a PR to add that for formatting code.

https://prettier.io/